lib/wallaroo/core/topology/runner.pony

```diff
@@ -555,7 +556,7 @@ class StateRunner[In: Any val, Out: Any val, S: State ref] is (Runner &
         let state_wrapper = _state_initializer.decode(_rb, _auth)?
         ifdef "checkpoint_trace" then
           @printf[I32]("Successfully imported key %s\n".cstring(),
-            key.cstring())
+            key.string().cstring())
```

Comment:
Does this need to be `HashableKey.string(key)` like the others?

Reply:
IIRC there aren't too many more of this kind of `String` assumption, less than a dozen. I may as well fix them now rather than wait until the day `Key`'s type does change.